apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/paidWorks.js

```diff
@@ -154,23 +162,36 @@ const generateTicketDocumentOfPaidWorks = async ({ task, baseAttrs, context, loc
             bic: get(invoices, '0.recipient.bic'),
         },
         totalInWords: {
-            // TODO: DOMA-10594 paidWorks with multi-currency support for moneyToWords
-            totalSum: moneyToWords(totalSum.toFixed(2), { locale, currencyCode }),
-            totalVAT: moneyToWords(totalVAT.toFixed(2), { locale, currencyCode }),
+            wholePartTotalSum: n2words(numberFormatByLocale
+                .formatToParts(totalSum)
+                .map(p => p.type === 'integer' ? p.value : '')
+                .join(''), { lang: locale }),
+            decimalPartTotalSum:  n2words(numberFormatByLocale
+                .formatToParts(totalSum)
+                .map(p => p.type === 'fraction' ? p.value : '')
+                .join(''), { lang: locale, feminine: true }),
+            wholePartTotalVAT: n2words(numberFormatByLocale
```

Comment:
naming-wise its best to specify entity first and then specify the type

TotalWholePart
TotalDecimalPart

VATWholePart
VATDecimalPart